packages/components/src/date-time/index.js

```diff
@@ -36,6 +37,8 @@ function DateTimePicker(
 		false
 	);
 
+	const [ showCalendarEvents, setShowCalendarEvents ] = useState( false );
```

Comment:
If we go with this solution, should we store this setting against the user?

Reply:
I've removed the component for now. I think we can handle it in a follow-up PR. What do you think.